docs/writing-rules/private-rules.md

```diff
@@ -0,0 +1,34 @@
+---
+append_help_link: true
```

Comment:
We don’t use this in the frontmatter anymore. (Sorry, the docs about the docs should be updated to reflect this.)

Instead, use `import MoreHelp from "/src/components/MoreHelp"` at the top of the document and then call it with `<MoreHelp />` at the bottom of the document. See the `pattern-examples.md` doc as an example.